modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/CustomDataSets.tsx

```diff
@@ -653,7 +659,7 @@ const CustomDataSets = ({
 		<div className="custom-data-sets data-sets">
 			<FrontendDataSet
 				{...FDS_DEFAULT_PROPS}
-				apiURL={API_URL.DATA_SETS}
+				apiURL={`${API_URL.DATA_SETS}?filter=not (externalReferenceCode in (${systemDataSetNames}))`}
```

Comment:
With this approach, filters, sorting, search and pagination will work on both pages. This would not be the case if we had to filter items manually before passing it in.

Reply:
We can simplify this after we add `system` flag, as [mentioned in Slack](https://liferay.slack.com/archives/C3JBR21HA/p1734612882534639?thread_ts=1734596315.410889&cid=C3JBR21HA).